backend/src/routes/analytics.py

```
@@ -0,0 +1,92 @@
+            "solBalance": wallet_info.get('sol_balance', 0) if wallet_info else 0
+        }
+
+        response_data = {
+            "stats": stats,
+            "topTokens": top_tokens,
+            "recentTrades": recent_trades,
+            "activePositions": active_positions
+        }
+
+        return success_response(data=response_data)
+    except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)